Update phpunit test phpunit/Controller/ContentReleaseStatusControllerTest.php

## Description
Test initially failed with multiple errors:

```
1) tests\phpunit\Controller\ContentReleaseStatusControllerTest::testContentReleaseStatusController with data set #0 (1690430400, 1690430400, 0, 'VA.gov last updated<br />toda...:00 am')
TypeError: Double\GuzzleHttp\Client\P27::get(): Return value must be of type Psr\Http\Message\ResponseInterface, null returned

/var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php:126
/var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php:89
/var/www/html/tests/phpunit/Controller/ContentReleaseStatusControllerTest.php:155
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestResult.php:728
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/TestRunner.php:651
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:144
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:97

2) tests\phpunit\Controller\ContentReleaseStatusControllerTest::testContentReleaseStatusController with data set #1 (1690430400, 1690483472, 0, 'VA.gov last updated<br />toda...:00 am')
TypeError: Double\GuzzleHttp\Client\P27::get(): Return value must be of type Psr\Http\Message\ResponseInterface, null returned

/var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php:126
/var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php:89
/var/www/html/tests/phpunit/Controller/ContentReleaseStatusControllerTest.php:155
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestResult.php:728
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/TestRunner.php:651
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:144
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:97
```

## Acceptance Criteria
- [ ] Tests pass.

### Team
Please check the team(s) that will do this work.

- [x] `CMS Team`
- [ ] `Public Websites`
- [ ] `Facilities`
- [ ] `User support`
- [ ] `Accelerated Publishing`

I updated the guzzle client to directly instantiate instead of using the service. Will need cleanup. Now tests fail instead of error. 
`edmunddunn@va-gov-cms-web:/var/www/html$ bin/phpunit \ --group unit \ --exclude-group disabled \ --coverage-text \ tests/phpunit
PHPUnit 9.6.10 by Sebastian Bergmann and contributors.

Warning:       XDEBUG_MODE=coverage or xdebug.mode=coverage has to be set

Testing /var/www/html/tests/phpunit
...........................................FFFFFFFFFFFFFFFFFFF.  63 / 397 ( 15%)
E.............................................................. 126 / 397 ( 31%)
............................................................... 189 / 397 ( 47%)
............................................................... 252 / 397 ( 63%)
............................................................... 315 / 397 ( 79%)
............................................................... 378 / 397 ( 95%)
...................                                             397 / 397 (100%)

Time: 10:14.073, Memory: 30.00 MB

There was 1 error:

1) test\phpunit\Deploy\DeployServiceTest::testCreate
Error: Call to undefined method test\phpunit\Deploy\DeployServiceTest::assertArrayEquals()

/var/www/html/tests/phpunit/Deploy/DeployServiceTest.php:52
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestResult.php:728
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/TestRunner.php:651
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:144
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:97

--

There were 19 failures:

1) tests\phpunit\Controller\ContentReleaseStatusControllerTest::testContentReleaseStatusController with data set #0 (1690430400, 1690430400, 0, 'VA.gov last updated<br />toda...:00 am')
assert($timestamp2 >= $timestamp1) in /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php:194

/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestResult.php:756
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/TestRunner.php:651
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:144
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:97

Caused by
AssertionError: assert($timestamp2 >= $timestamp1) in /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php:194
Stack trace:
#0 /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php(194): assert(false, 'assert($timesta...')
#1 /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php(165): Drupal\va_gov_backend\Controller\ContentReleaseStatusController->getDifferenceInDays(1690482109, 1690430400)
#2 /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php(148): Drupal\va_gov_backend\Controller\ContentReleaseStatusController->getDifferenceInDaysMessage(1690482109, 1690430400)
#3 /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php(92): Drupal\va_gov_backend\Controller\ContentReleaseStatusController->getLastReleaseMessage(1690482109, 1690430400)
#4 /var/www/html/tests/phpunit/Controller/ContentReleaseStatusControllerTest.php(155): Drupal\va_gov_backend\Controller\ContentReleaseStatusController->getLastReleaseStatus(1690430400)
#5 /var/www/html/docroot/sites/simpletest/TestCase.php(1610): tests\phpunit\Controller\ContentReleaseStatusControllerTest->testContentReleaseStatusController(1690430400, 1690430400, 0, 'VA.gov last upd...')
#6 /var/www/html/docroot/sites/simpletest/TestCase.php(1216): PHPUnit\Framework\TestCase->runTest()
#7 /var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#8 /var/www/html/docroot/sites/simpletest/TestCase.php(966): PHPUnit\Framework\TestResult->run(Object(tests\phpunit\Controller\ContentReleaseStatusControllerTest))
#9 /var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php(684): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#10 /var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php(684): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 /var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php(684): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#12 /var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/TestRunner.php(651): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#13 /var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#14 /var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#15 /var/www/html/docroot/vendor/phpunit/phpunit/phpunit(107): PHPUnit\TextUI\Command::main()
#16 /var/www/html/bin/phpunit(122): include('/var/www/html/d...')
#17 {main}

2) tests\phpunit\Controller\ContentReleaseStatusControllerTest::testContentReleaseStatusController with data set #1 (1690430400, 1690483913, 0, 'VA.gov last updated<br />toda...:00 am')
Failed asserting that Drupal\Core\StringTranslation\TranslatableMarkup Object &00000000000034f30000000000000000 (
    'string' => 'VA.gov last updated<br />@release_message'
    'arguments' => Array &0 (
        '@release_message' => 'today at 02:21 pm'
    )
    'translatedMarkup' => 'VA.gov last updated<br />@release_message'
    'options' => Array &1 ()
    'stringTranslation' => Double\TranslationInterface\P17 Object &000000000000356a0000000000000000 (
        'objectProphecyClosure' => Closure Object &00000000000035620000000000000000 (
            0 => Closure Object &00000000000035620000000000000000
        )
    )
) matches expected 'VA.gov last updated<br />today at 12:00 am'.

/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/Constraint/Equality/IsEqual.php:96
/var/www/html/tests/phpunit/Controller/ContentReleaseStatusControllerTest.php:157
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestResult.php:728
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/TestRunner.php:651
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:144
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:97

3) tests\phpunit\Controller\ContentReleaseStatusControllerTest::testContentReleaseStatusController with data set #2 (1690483913, 1690483913, 0, 'VA.gov last updated<br />toda...:51 pm')
Failed asserting that Drupal\Core\StringTranslation\TranslatableMarkup Object &00000000000036a00000000000000000 (
    'string' => 'VA.gov last updated<br />@release_message'
    'arguments' => Array &0 (
        '@release_message' => 'today at 02:21 pm'
    )
    'translatedMarkup' => 'VA.gov last updated<br />@release_message'
    'options' => Array &1 ()
    'stringTranslation' => Double\TranslationInterface\P17 Object &00000000000036700000000000000000 (
        'objectProphecyClosure' => Closure Object &000000000000366f0000000000000000 (
            0 => Closure Object &000000000000366f0000000000000000
        )
    )
) matches expected 'VA.gov last updated<br />today at 02:51 pm'.

/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/Constraint/Equality/IsEqual.php:96
/var/www/html/tests/phpunit/Controller/ContentReleaseStatusControllerTest.php:157
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestResult.php:728
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/TestRunner.php:651
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:144
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:97

4) tests\phpunit\Controller\ContentReleaseStatusControllerTest::testContentReleaseStatusController with data set #3 (1690430395, 1690483913, 1, 'VA.gov last updated<br />yest...:59 pm')
Failed asserting that Drupal\Core\StringTranslation\TranslatableMarkup Object &00000000000037250000000000000000 (
    'string' => 'VA.gov last updated<br />@release_message'
    'arguments' => Array &0 (
        '@release_message' => 'today at 02:21 pm'
    )
    'translatedMarkup' => 'VA.gov last updated<br />@release_message'
    'options' => Array &1 ()
    'stringTranslation' => Double\TranslationInterface\P17 Object &00000000000036f50000000000000000 (
        'objectProphecyClosure' => Closure Object &00000000000036f40000000000000000 (
            0 => Closure Object &00000000000036f40000000000000000
        )
    )
) matches expected 'VA.gov last updated<br />yesterday at 11:59 pm'.

/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/Constraint/Equality/IsEqual.php:96
/var/www/html/tests/phpunit/Controller/ContentReleaseStatusControllerTest.php:157
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestResult.php:728
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/TestRunner.php:651
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:144
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:97

5) tests\phpunit\Controller\ContentReleaseStatusControllerTest::testContentReleaseStatusController with data set #4 (1690344000, 1690344000, 0, 'VA.gov last updated<br />toda...:00 am')
assert($timestamp2 >= $timestamp1) in /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php:194

/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestResult.php:756
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/TestRunner.php:651
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:144
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:97

Caused by
AssertionError: assert($timestamp2 >= $timestamp1) in /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php:194
Stack trace:
#0 /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php(194): assert(false, 'assert($timesta...')
#1 /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php(165): Drupal\va_gov_backend\Controller\ContentReleaseStatusController->getDifferenceInDays(1690482109, 1690344000)
#2 /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php(148): Drupal\va_gov_backend\Controller\ContentReleaseStatusController->getDifferenceInDaysMessage(1690482109, 1690344000)
#3 /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php(92): Drupal\va_gov_backend\Controller\ContentReleaseStatusController->getLastReleaseMessage(1690482109, 1690344000)
#4 /var/www/html/tests/phpunit/Controller/ContentReleaseStatusControllerTest.php(155): Drupal\va_gov_backend\Controller\ContentReleaseStatusController->getLastReleaseStatus(1690344000)
#5 /var/www/html/docroot/sites/simpletest/TestCase.php(1610): tests\phpunit\Controller\ContentReleaseStatusControllerTest->testContentReleaseStatusController(1690344000, 1690344000, 0, 'VA.gov last upd...')
#6 /var/www/html/docroot/sites/simpletest/TestCase.php(1216): PHPUnit\Framework\TestCase->runTest()
#7 /var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#8 /var/www/html/docroot/sites/simpletest/TestCase.php(966): PHPUnit\Framework\TestResult->run(Object(tests\phpunit\Controller\ContentReleaseStatusControllerTest))
#9 /var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php(684): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#10 /var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php(684): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 /var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php(684): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#12 /var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/TestRunner.php(651): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#13 /var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#14 /var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#15 /var/www/html/docroot/vendor/phpunit/phpunit/phpunit(107): PHPUnit\TextUI\Command::main()
#16 /var/www/html/bin/phpunit(122): include('/var/www/html/d...')
#17 {main}

6) tests\phpunit\Controller\ContentReleaseStatusControllerTest::testContentReleaseStatusController with data set #5 (1690344000, 1690430400, 1, 'VA.gov last updated<br />yest...:00 am')
assert($timestamp2 >= $timestamp1) in /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php:194

/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestResult.php:756
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/TestRunner.php:651
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:144
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:97

Caused by
AssertionError: assert($timestamp2 >= $timestamp1) in /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php:194
Stack trace:
#0 /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php(194): assert(false, 'assert($timesta...')
#1 /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php(165): Drupal\va_gov_backend\Controller\ContentReleaseStatusController->getDifferenceInDays(1690482109, 1690430400)
#2 /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php(148): Drupal\va_gov_backend\Controller\ContentReleaseStatusController->getDifferenceInDaysMessage(1690482109, 1690430400)
#3 /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php(92): Drupal\va_gov_backend\Controller\ContentReleaseStatusController->getLastReleaseMessage(1690482109, 1690430400)
#4 /var/www/html/tests/phpunit/Controller/ContentReleaseStatusControllerTest.php(155): Drupal\va_gov_backend\Controller\ContentReleaseStatusController->getLastReleaseStatus(1690430400)
#5 /var/www/html/docroot/sites/simpletest/TestCase.php(1610): tests\phpunit\Controller\ContentReleaseStatusControllerTest->testContentReleaseStatusController(1690344000, 1690430400, 1, 'VA.gov last upd...')
#6 /var/www/html/docroot/sites/simpletest/TestCase.php(1216): PHPUnit\Framework\TestCase->runTest()
#7 /var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#8 /var/www/html/docroot/sites/simpletest/TestCase.php(966): PHPUnit\Framework\TestResult->run(Object(tests\phpunit\Controller\ContentReleaseStatusControllerTest))
#9 /var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php(684): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#10 /var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php(684): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 /var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php(684): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#12 /var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/TestRunner.php(651): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#13 /var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#14 /var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#15 /var/www/html/docroot/vendor/phpunit/phpunit/phpunit(107): PHPUnit\TextUI\Command::main()
#16 /var/www/html/bin/phpunit(122): include('/var/www/html/d...')
#17 {main}

7) tests\phpunit\Controller\ContentReleaseStatusControllerTest::testContentReleaseStatusController with data set #6 (1690344000, 1690483913, 1, 'VA.gov last updated<br />yest...:00 am')
Failed asserting that Drupal\Core\StringTranslation\TranslatableMarkup Object &000000000000389e0000000000000000 (
    'string' => 'VA.gov last updated<br />@release_message'
    'arguments' => Array &0 (
        '@release_message' => 'today at 02:21 pm'
    )
    'translatedMarkup' => 'VA.gov last updated<br />@release_message'
    'options' => Array &1 ()
    'stringTranslation' => Double\TranslationInterface\P17 Object &000000000000386e0000000000000000 (
        'objectProphecyClosure' => Closure Object &000000000000386d0000000000000000 (
            0 => Closure Object &000000000000386d0000000000000000
        )
    )
) matches expected 'VA.gov last updated<br />yesterday at 12:00 am'.

/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/Constraint/Equality/IsEqual.php:96
/var/www/html/tests/phpunit/Controller/ContentReleaseStatusControllerTest.php:157
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestResult.php:728
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/TestRunner.php:651
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:144
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:97

8) tests\phpunit\Controller\ContentReleaseStatusControllerTest::testContentReleaseStatusController with data set #7 (1690394700, 1690483913, 1, 'VA.gov last updated<br />yest...:05 pm')
Failed asserting that Drupal\Core\StringTranslation\TranslatableMarkup Object &00000000000039230000000000000000 (
    'string' => 'VA.gov last updated<br />@release_message'
    'arguments' => Array &0 (
        '@release_message' => 'today at 02:21 pm'
    )
    'translatedMarkup' => 'VA.gov last updated<br />@release_message'
    'options' => Array &1 ()
    'stringTranslation' => Double\TranslationInterface\P17 Object &00000000000038f30000000000000000 (
        'objectProphecyClosure' => Closure Object &00000000000038f20000000000000000 (
            0 => Closure Object &00000000000038f20000000000000000
        )
    )
) matches expected 'VA.gov last updated<br />yesterday at 02:05 pm'.

/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/Constraint/Equality/IsEqual.php:96
/var/www/html/tests/phpunit/Controller/ContentReleaseStatusControllerTest.php:157
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestResult.php:728
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/TestRunner.php:651
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:144
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:97

9) tests\phpunit\Controller\ContentReleaseStatusControllerTest::testContentReleaseStatusController with data set #8 (1690344000, 1690344000, 0, 'VA.gov last updated<br />toda...:00 am')
assert($timestamp2 >= $timestamp1) in /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php:194

/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestResult.php:756
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/TestRunner.php:651
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:144
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:97

Caused by
AssertionError: assert($timestamp2 >= $timestamp1) in /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php:194
Stack trace:
#0 /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php(194): assert(false, 'assert($timesta...')
#1 /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php(165): Drupal\va_gov_backend\Controller\ContentReleaseStatusController->getDifferenceInDays(1690482109, 1690344000)
#2 /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php(148): Drupal\va_gov_backend\Controller\ContentReleaseStatusController->getDifferenceInDaysMessage(1690482109, 1690344000)
#3 /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php(92): Drupal\va_gov_backend\Controller\ContentReleaseStatusController->getLastReleaseMessage(1690482109, 1690344000)
#4 /var/www/html/tests/phpunit/Controller/ContentReleaseStatusControllerTest.php(155): Drupal\va_gov_backend\Controller\ContentReleaseStatusController->getLastReleaseStatus(1690344000)
#5 /var/www/html/docroot/sites/simpletest/TestCase.php(1610): tests\phpunit\Controller\ContentReleaseStatusControllerTest->testContentReleaseStatusController(1690344000, 1690344000, 0, 'VA.gov last upd...')
#6 /var/www/html/docroot/sites/simpletest/TestCase.php(1216): PHPUnit\Framework\TestCase->runTest()
#7 /var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#8 /var/www/html/docroot/sites/simpletest/TestCase.php(966): PHPUnit\Framework\TestResult->run(Object(tests\phpunit\Controller\ContentReleaseStatusControllerTest))
#9 /var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php(684): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#10 /var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php(684): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 /var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php(684): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#12 /var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/TestRunner.php(651): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#13 /var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#14 /var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#15 /var/www/html/docroot/vendor/phpunit/phpunit/phpunit(107): PHPUnit\TextUI\Command::main()
#16 /var/www/html/bin/phpunit(122): include('/var/www/html/d...')
#17 {main}

10) tests\phpunit\Controller\ContentReleaseStatusControllerTest::testContentReleaseStatusController with data set #9 (1690344000, 1690344000, 0, 'VA.gov last updated<br />toda...:00 am')
assert($timestamp2 >= $timestamp1) in /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php:194

/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestResult.php:756
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/TestRunner.php:651
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:144
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:97

Caused by
AssertionError: assert($timestamp2 >= $timestamp1) in /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php:194
Stack trace:
#0 /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php(194): assert(false, 'assert($timesta...')
#1 /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php(165): Drupal\va_gov_backend\Controller\ContentReleaseStatusController->getDifferenceInDays(1690482109, 1690344000)
#2 /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php(148): Drupal\va_gov_backend\Controller\ContentReleaseStatusController->getDifferenceInDaysMessage(1690482109, 1690344000)
#3 /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php(92): Drupal\va_gov_backend\Controller\ContentReleaseStatusController->getLastReleaseMessage(1690482109, 1690344000)
#4 /var/www/html/tests/phpunit/Controller/ContentReleaseStatusControllerTest.php(155): Drupal\va_gov_backend\Controller\ContentReleaseStatusController->getLastReleaseStatus(1690344000)
#5 /var/www/html/docroot/sites/simpletest/TestCase.php(1610): tests\phpunit\Controller\ContentReleaseStatusControllerTest->testContentReleaseStatusController(1690344000, 1690344000, 0, 'VA.gov last upd...')
#6 /var/www/html/docroot/sites/simpletest/TestCase.php(1216): PHPUnit\Framework\TestCase->runTest()
#7 /var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#8 /var/www/html/docroot/sites/simpletest/TestCase.php(966): PHPUnit\Framework\TestResult->run(Object(tests\phpunit\Controller\ContentReleaseStatusControllerTest))
#9 /var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php(684): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#10 /var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php(684): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 /var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php(684): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#12 /var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/TestRunner.php(651): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#13 /var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#14 /var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#15 /var/www/html/docroot/vendor/phpunit/phpunit/phpunit(107): PHPUnit\TextUI\Command::main()
#16 /var/www/html/bin/phpunit(122): include('/var/www/html/d...')
#17 {main}

11) tests\phpunit\Controller\ContentReleaseStatusControllerTest::testContentReleaseStatusController with data set #10 (1690344000, 1690483913, 1, 'VA.gov last updated<br />yest...:00 am')
Failed asserting that Drupal\Core\StringTranslation\TranslatableMarkup Object &0000000000003a9c0000000000000000 (
    'string' => 'VA.gov last updated<br />@release_message'
    'arguments' => Array &0 (
        '@release_message' => 'today at 02:21 pm'
    )
    'translatedMarkup' => 'VA.gov last updated<br />@release_message'
    'options' => Array &1 ()
    'stringTranslation' => Double\TranslationInterface\P17 Object &0000000000003a6c0000000000000000 (
        'objectProphecyClosure' => Closure Object &0000000000003a6b0000000000000000 (
            0 => Closure Object &0000000000003a6b0000000000000000
        )
    )
) matches expected 'VA.gov last updated<br />yesterday at 12:00 am'.

/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/Constraint/Equality/IsEqual.php:96
/var/www/html/tests/phpunit/Controller/ContentReleaseStatusControllerTest.php:157
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestResult.php:728
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/TestRunner.php:651
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:144
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:97

12) tests\phpunit\Controller\ContentReleaseStatusControllerTest::testContentReleaseStatusController with data set #11 (1690344000, 1690430400, 1, 'VA.gov last updated<br />yest...:00 am')
assert($timestamp2 >= $timestamp1) in /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php:194

/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestResult.php:756
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/TestRunner.php:651
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:144
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:97

Caused by
AssertionError: assert($timestamp2 >= $timestamp1) in /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php:194
Stack trace:
#0 /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php(194): assert(false, 'assert($timesta...')
#1 /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php(165): Drupal\va_gov_backend\Controller\ContentReleaseStatusController->getDifferenceInDays(1690482109, 1690430400)
#2 /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php(148): Drupal\va_gov_backend\Controller\ContentReleaseStatusController->getDifferenceInDaysMessage(1690482109, 1690430400)
#3 /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php(92): Drupal\va_gov_backend\Controller\ContentReleaseStatusController->getLastReleaseMessage(1690482109, 1690430400)
#4 /var/www/html/tests/phpunit/Controller/ContentReleaseStatusControllerTest.php(155): Drupal\va_gov_backend\Controller\ContentReleaseStatusController->getLastReleaseStatus(1690430400)
#5 /var/www/html/docroot/sites/simpletest/TestCase.php(1610): tests\phpunit\Controller\ContentReleaseStatusControllerTest->testContentReleaseStatusController(1690344000, 1690430400, 1, 'VA.gov last upd...')
#6 /var/www/html/docroot/sites/simpletest/TestCase.php(1216): PHPUnit\Framework\TestCase->runTest()
#7 /var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#8 /var/www/html/docroot/sites/simpletest/TestCase.php(966): PHPUnit\Framework\TestResult->run(Object(tests\phpunit\Controller\ContentReleaseStatusControllerTest))
#9 /var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php(684): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#10 /var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php(684): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 /var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php(684): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#12 /var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/TestRunner.php(651): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#13 /var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#14 /var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#15 /var/www/html/docroot/vendor/phpunit/phpunit/phpunit(107): PHPUnit\TextUI\Command::main()
#16 /var/www/html/bin/phpunit(122): include('/var/www/html/d...')
#17 {main}

13) tests\phpunit\Controller\ContentReleaseStatusControllerTest::testContentReleaseStatusController with data set #12 (1690257600, 1690430400, 2, 'VA.gov last updated<br />2 da...:00 am')
assert($timestamp2 >= $timestamp1) in /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php:194

/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestResult.php:756
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/TestRunner.php:651
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:144
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:97

Caused by
AssertionError: assert($timestamp2 >= $timestamp1) in /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php:194
Stack trace:
#0 /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php(194): assert(false, 'assert($timesta...')
#1 /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php(165): Drupal\va_gov_backend\Controller\ContentReleaseStatusController->getDifferenceInDays(1690482109, 1690430400)
#2 /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php(148): Drupal\va_gov_backend\Controller\ContentReleaseStatusController->getDifferenceInDaysMessage(1690482109, 1690430400)
#3 /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php(92): Drupal\va_gov_backend\Controller\ContentReleaseStatusController->getLastReleaseMessage(1690482109, 1690430400)
#4 /var/www/html/tests/phpunit/Controller/ContentReleaseStatusControllerTest.php(155): Drupal\va_gov_backend\Controller\ContentReleaseStatusController->getLastReleaseStatus(1690430400)
#5 /var/www/html/docroot/sites/simpletest/TestCase.php(1610): tests\phpunit\Controller\ContentReleaseStatusControllerTest->testContentReleaseStatusController(1690257600, 1690430400, 2, 'VA.gov last upd...')
#6 /var/www/html/docroot/sites/simpletest/TestCase.php(1216): PHPUnit\Framework\TestCase->runTest()
#7 /var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#8 /var/www/html/docroot/sites/simpletest/TestCase.php(966): PHPUnit\Framework\TestResult->run(Object(tests\phpunit\Controller\ContentReleaseStatusControllerTest))
#9 /var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php(684): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#10 /var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php(684): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 /var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php(684): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#12 /var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/TestRunner.php(651): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#13 /var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#14 /var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#15 /var/www/html/docroot/vendor/phpunit/phpunit/phpunit(107): PHPUnit\TextUI\Command::main()
#16 /var/www/html/bin/phpunit(122): include('/var/www/html/d...')
#17 {main}

14) tests\phpunit\Controller\ContentReleaseStatusControllerTest::testContentReleaseStatusController with data set #13 (1690257600, 1690483913, 2, 'VA.gov last updated<br />2 da...:00 am')
Failed asserting that Drupal\Core\StringTranslation\TranslatableMarkup Object &0000000000003c150000000000000000 (
    'string' => 'VA.gov last updated<br />@release_message'
    'arguments' => Array &0 (
        '@release_message' => 'today at 02:21 pm'
    )
    'translatedMarkup' => 'VA.gov last updated<br />@release_message'
    'options' => Array &1 ()
    'stringTranslation' => Double\TranslationInterface\P17 Object &0000000000003be50000000000000000 (
        'objectProphecyClosure' => Closure Object &0000000000003be40000000000000000 (
            0 => Closure Object &0000000000003be40000000000000000
        )
    )
) matches expected 'VA.gov last updated<br />2 days ago at 12:00 am'.

/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/Constraint/Equality/IsEqual.php:96
/var/www/html/tests/phpunit/Controller/ContentReleaseStatusControllerTest.php:157
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestResult.php:728
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/TestRunner.php:651
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:144
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:97

15) tests\phpunit\Controller\ContentReleaseStatusControllerTest::testContentReleaseStatusController with data set #14 (1690171200, 1690430400, 3, 'VA.gov last updated<br />3 da...:00 am')
assert($timestamp2 >= $timestamp1) in /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php:194

/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestResult.php:756
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/TestRunner.php:651
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:144
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:97

Caused by
AssertionError: assert($timestamp2 >= $timestamp1) in /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php:194
Stack trace:
#0 /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php(194): assert(false, 'assert($timesta...')
#1 /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php(165): Drupal\va_gov_backend\Controller\ContentReleaseStatusController->getDifferenceInDays(1690482109, 1690430400)
#2 /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php(148): Drupal\va_gov_backend\Controller\ContentReleaseStatusController->getDifferenceInDaysMessage(1690482109, 1690430400)
#3 /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php(92): Drupal\va_gov_backend\Controller\ContentReleaseStatusController->getLastReleaseMessage(1690482109, 1690430400)
#4 /var/www/html/tests/phpunit/Controller/ContentReleaseStatusControllerTest.php(155): Drupal\va_gov_backend\Controller\ContentReleaseStatusController->getLastReleaseStatus(1690430400)
#5 /var/www/html/docroot/sites/simpletest/TestCase.php(1610): tests\phpunit\Controller\ContentReleaseStatusControllerTest->testContentReleaseStatusController(1690171200, 1690430400, 3, 'VA.gov last upd...')
#6 /var/www/html/docroot/sites/simpletest/TestCase.php(1216): PHPUnit\Framework\TestCase->runTest()
#7 /var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#8 /var/www/html/docroot/sites/simpletest/TestCase.php(966): PHPUnit\Framework\TestResult->run(Object(tests\phpunit\Controller\ContentReleaseStatusControllerTest))
#9 /var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php(684): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#10 /var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php(684): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 /var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php(684): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#12 /var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/TestRunner.php(651): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#13 /var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#14 /var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#15 /var/www/html/docroot/vendor/phpunit/phpunit/phpunit(107): PHPUnit\TextUI\Command::main()
#16 /var/www/html/bin/phpunit(122): include('/var/www/html/d...')
#17 {main}

16) tests\phpunit\Controller\ContentReleaseStatusControllerTest::testContentReleaseStatusController with data set #15 (1690171200, 1690483913, 3, 'VA.gov last updated<br />3 da...:00 am')
Failed asserting that Drupal\Core\StringTranslation\TranslatableMarkup Object &0000000000003d140000000000000000 (
    'string' => 'VA.gov last updated<br />@release_message'
    'arguments' => Array &0 (
        '@release_message' => 'today at 02:21 pm'
    )
    'translatedMarkup' => 'VA.gov last updated<br />@release_message'
    'options' => Array &1 ()
    'stringTranslation' => Double\TranslationInterface\P17 Object &0000000000003ce40000000000000000 (
        'objectProphecyClosure' => Closure Object &0000000000003ce30000000000000000 (
            0 => Closure Object &0000000000003ce30000000000000000
        )
    )
) matches expected 'VA.gov last updated<br />3 days ago at 12:00 am'.

/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/Constraint/Equality/IsEqual.php:96
/var/www/html/tests/phpunit/Controller/ContentReleaseStatusControllerTest.php:157
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestResult.php:728
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/TestRunner.php:651
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:144
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:97

17) tests\phpunit\Controller\ContentReleaseStatusControllerTest::testContentReleaseStatusController with data set #16 (1690084800, 1690430400, 4, 'VA.gov last updated<br />4 da...:00 am')
assert($timestamp2 >= $timestamp1) in /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php:194

/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestResult.php:756
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/TestRunner.php:651
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:144
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:97

Caused by
AssertionError: assert($timestamp2 >= $timestamp1) in /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php:194
Stack trace:
#0 /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php(194): assert(false, 'assert($timesta...')
#1 /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php(165): Drupal\va_gov_backend\Controller\ContentReleaseStatusController->getDifferenceInDays(1690482109, 1690430400)
#2 /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php(148): Drupal\va_gov_backend\Controller\ContentReleaseStatusController->getDifferenceInDaysMessage(1690482109, 1690430400)
#3 /var/www/html/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php(92): Drupal\va_gov_backend\Controller\ContentReleaseStatusController->getLastReleaseMessage(1690482109, 1690430400)
#4 /var/www/html/tests/phpunit/Controller/ContentReleaseStatusControllerTest.php(155): Drupal\va_gov_backend\Controller\ContentReleaseStatusController->getLastReleaseStatus(1690430400)
#5 /var/www/html/docroot/sites/simpletest/TestCase.php(1610): tests\phpunit\Controller\ContentReleaseStatusControllerTest->testContentReleaseStatusController(1690084800, 1690430400, 4, 'VA.gov last upd...')
#6 /var/www/html/docroot/sites/simpletest/TestCase.php(1216): PHPUnit\Framework\TestCase->runTest()
#7 /var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestResult.php(728): PHPUnit\Framework\TestCase->runBare()
#8 /var/www/html/docroot/sites/simpletest/TestCase.php(966): PHPUnit\Framework\TestResult->run(Object(tests\phpunit\Controller\ContentReleaseStatusControllerTest))
#9 /var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php(684): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#10 /var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php(684): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#11 /var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php(684): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#12 /var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/TestRunner.php(651): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#13 /var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php(144): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#14 /var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php(97): PHPUnit\TextUI\Command->run(Array, true)
#15 /var/www/html/docroot/vendor/phpunit/phpunit/phpunit(107): PHPUnit\TextUI\Command::main()
#16 /var/www/html/bin/phpunit(122): include('/var/www/html/d...')
#17 {main}

18) tests\phpunit\Controller\ContentReleaseStatusControllerTest::testContentReleaseStatusController with data set #17 (1690084800, 1690483913, 4, 'VA.gov last updated<br />4 da...:00 am')
Failed asserting that Drupal\Core\StringTranslation\TranslatableMarkup Object &0000000000003e130000000000000000 (
    'string' => 'VA.gov last updated<br />@release_message'
    'arguments' => Array &0 (
        '@release_message' => 'today at 02:21 pm'
    )
    'translatedMarkup' => 'VA.gov last updated<br />@release_message'
    'options' => Array &1 ()
    'stringTranslation' => Double\TranslationInterface\P17 Object &0000000000003de30000000000000000 (
        'objectProphecyClosure' => Closure Object &0000000000003de20000000000000000 (
            0 => Closure Object &0000000000003de20000000000000000
        )
    )
) matches expected 'VA.gov last updated<br />4 days ago at 12:00 am'.

/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/Constraint/Equality/IsEqual.php:96
/var/www/html/tests/phpunit/Controller/ContentReleaseStatusControllerTest.php:157
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestResult.php:728
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/TestRunner.php:651
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:144
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:97

19) tests\phpunit\Controller\ContentReleaseStatusControllerTest::testContentReleaseStatusController with data set #18 (1690123080, 1690483913, 4, 'VA.gov last updated<br />4 da...:38 am')
Failed asserting that Drupal\Core\StringTranslation\TranslatableMarkup Object &0000000000003e980000000000000000 (
    'string' => 'VA.gov last updated<br />@release_message'
    'arguments' => Array &0 (
        '@release_message' => 'today at 02:21 pm'
    )
    'translatedMarkup' => 'VA.gov last updated<br />@release_message'
    'options' => Array &1 ()
    'stringTranslation' => Double\TranslationInterface\P17 Object &0000000000003e680000000000000000 (
        'objectProphecyClosure' => Closure Object &0000000000003e670000000000000000 (
            0 => Closure Object &0000000000003e670000000000000000
        )
    )
) matches expected 'VA.gov last updated<br />4 days ago at 10:38 am'.

/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/Constraint/Equality/IsEqual.php:96
/var/www/html/tests/phpunit/Controller/ContentReleaseStatusControllerTest.php:157
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestResult.php:728
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/Framework/TestSuite.php:684
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/TestRunner.php:651
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:144
/var/www/html/docroot/vendor/phpunit/phpunit/src/TextUI/Command.php:97

ERRORS!
Tests: 397, Assertions: 833, Errors: 1, Failures: 19.`

--- a/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php
+++ b/docroot/modules/custom/va_gov_backend/src/Controller/ContentReleaseStatusController.php
@@ -123,7 +123,7 @@ class ContentReleaseStatusController extends ControllerBase {
    */
   public function getLastReleaseTimestamp(): int {
     /** @var \Psr\Http\Message\ResponseInterface $response */
-    $response = $this->httpClient->request('GET', 'https://www.va.gov/BUILD.txt');
+    $response = $this->httpClient->get('https://www.va.gov/BUILD.txt');
     if (preg_match('/BUILDTIME=([0-9]*)/', $response->getBody(), $matches)) {
       return $matches[1];
     }

--- a/tests/phpunit/Controller/ContentReleaseStatusControllerTest.php
+++ b/tests/phpunit/Controller/ContentReleaseStatusControllerTest.php
@@ -92,7 +92,7 @@ class ContentReleaseStatusControllerTest extends VaGovUnitTestBase {
     $responseProphecy = $this->prophesize(Response::CLASS);
     $responseProphecy->getBody()->willReturn($responseBody);
 
-    $prophecy->request(Argument::type('string'), Argument::type('string'))->willReturn($responseProphecy->reveal());
+    $prophecy->get(Argument::type('string'))->willReturn($responseProphecy->reveal());
 
     return $prophecy->reveal();
   }

